packages/kotti-user-menu/src/UserMenu.vue

```diff
@@ -126,3 +111,131 @@ export default {
 	},
 }
 </script>
+<style lang="scss">
+:root {
+	--user-menu-border: var(--primary-80);
+	--user-menu-background-active: var(--primary-70);
+	--user-menu-background: var(--primary-60);
+	--user-menu-color: var(--primary-10);
+}
+
+.user-info {
+	position: relative;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	padding: 0.4rem;
+	margin: -0.4rem;
+	line-height: 1;
+	&--narrow {
+		width: 2.4rem;
+	}
+	&--narrow-wide {
+		position: absolute;
+		bottom: 0.8rem;
+		left: 0.8rem;
+		width: 11.2rem;
+	}
+	&:hover {
+		cursor: pointer;
+		background: var(--user-menu-background);
+		border-radius: 0.2rem;
+	}
+}
+.user-info-avatar {
+	z-index: 2;
+	flex-grow: 0;
+}
+.user-info-text {
+	z-index: 2;
+	width: 100%;
+	margin-left: 0.4rem;
```

Comment:
@tomquinonero I still see a lot of spacing values defined in the old way, do you not want to update those as well?

Reply:
@mvangent Yes, ideally, but I tried to keep the scope tight on this one!